source/brailleDisplayDrivers/handyTech.py

```diff
@@ -656,7 +709,7 @@ def registerAutomaticDetection(cls, driverRegistrar: bdDetect.DriverRegistrar):
 			"Braillino BL",
 			"Braille Wave BW",
 			"Easy Braille EBR",
-			"Activator AC",

```

Comment:
Is this removal intentional?

Reply:
Yes. The intent is to match Activator AC4/serial, Activator Pro 64/serial and Activator Pro 80/serial with a single condition.